src/factories/DisputeBondTokenFactory.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.4.11;
```

Comment:
We should pick a compiler version and use it consistently throughout the project.  I suspect we'll need to bump to `0.4.14` to pick up a bugfix I'm waiting on for circular imports.  I have been using `0.4.13` in the meantime.